src/contracts.js

```diff
@@ -24,17 +25,74 @@ const STAKE_ABI = [
   },
 ]
 
-const STAKE_ADDR = '0x7e16016df8c3d0a944cf568309b4214ab9856bee'
+const RESOLVER_ABI = [
+  {
+    "constant": true,
+    "inputs": [
+      {
+        "name": "node",
+        "type": "bytes32"
+      }
+    ],
+    "name": "addr",
+    "outputs": [
+      {
+        "name": "ret",
+        "type": "address"
+      }
+    ],
+    "payable": false,
+    "type": "function"
+  },
+]
+
+const INFURA_URLS = {
+  ROPSTEN: 'https://ropsten.infura.io/KWLG1YOMaYgl4wiFlcJv',
+  RINKEBY: 'https://rinkeby.infura.io/KWLG1YOMaYgl4wiFlcJv',
+  MAINNET: 'https://mainnet.infura.io/KWLG1YOMaYgl4wiFlcJv',
```

Comment:
We should remove these API keys and wipe them from git history.

Alos, it would be better to pass an eth RPC endpoint when starting onyx-server so that you could use a local node (or some other RPC API you have access to) to resolve these instead depending on a specific 3rd party node to be up.